cloudinary/api.py

```diff
@@ -510,3 +511,42 @@ def __delete_resource_params(options, **params):
              **only(options, "keep_original", "next_cursor", "invalidate"))
     p.update(params)
     return p
+
+
+def verify_api_response_signature(public_id, version, signature):
```

Comment:
I think these two should be moved out of api.py to a separate file, similar to PHP implementation. these are separate helper methods not that related to the API itself